driver-core/src/main/java/com/datastax/driver/core/querybuilder/BuiltStatement.java

```diff
@@ -293,25 +311,6 @@ public Boolean isIdempotent() {
         return !hasNonIdempotentOps();
     }
 
-    @Override
-    public String toString() {
-        try {
-            if (forceNoValues)
-                return getQueryString();
-            // 1) try first with all values inlined (will not work if some values require custom codecs,
-            // or if the required codecs are registered in a different CodecRegistry instance than the default one)
-            return maybeAddSemicolon(buildQueryString(null, CodecRegistry.DEFAULT_INSTANCE)).toString();
```

Comment:
We could even see this as a new advantage: now the query string included in the `toString()` result is the very same query string that would be executed if the statement was passed to `session.execute(Statement)`.